plugins/org.eclipse.n4js.transpiler.es/src/org/eclipse/n4js/transpiler/es/EcmaScriptSubGenerator.java

```diff
@@ -123,8 +122,10 @@ protected void internalDoGenerate(Resource resource, GeneratorOption[] options,
 			// the next two variables store the navigation-prefix to get to the sources
 			final Path relativeNavigationToSrc = calculateNavigationFromOutputToSourcePath(fsa, getCompilerID(),
 					resourceCasted);
-			final Path explicitNavigationToSrc = Paths.get("/sources");
-			boolean useExplicitSourceRef = true; // true use explicitNavigationToSrc | false use relativeNavigationToSrc
+			// TODO what's that?
+			// final Path explicitNavigationToSrc = Paths.get("/sources");
+			boolean useExplicitSourceRef = false; // true use explicitNavigationToSrc | false use
+													// relativeNavigationToSrc
```

Comment:
if those are not needed, they should be removed

Reply:
removed that. In the current version, path is always relative in the source map.